DocAggregatorDotnetCore/TomAggregator.cs

```diff
@@ -38,15 +33,22 @@ public void DoSomeWork()
 
             Directory.CreateDirectory(outputDirectoryPath);
 
-            //Console.WriteLine("New folders to go in: " + outputDirectoryPath);
+            //make array of property refs
+            var ctx = new RiggsTestContext();
+            var listOfDocs = ctx.PropDocs.Select(p => p).ToList();
+            var docsArray = listOfDocs.ToArray();
```

Comment:
I believe you can remove this line as you can foreach over your listOfDocs as it's a list.

`foreach` works on anything that implements the IEnumerable interface, which List does.